subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -593,8 +593,13 @@ static int cmd_wifi_ps(const struct shell *sh, size_t argc, char *argv[])
 		shell_fprintf(sh, SHELL_NORMAL, "PS wake up mode: %s\n",
 				config.ps_params.wakeup_mode ? "Listen interval" : "DTIM");
 
-		shell_fprintf(sh, SHELL_NORMAL, "PS timeout: %d ms\n",
-				config.ps_params.timeout_ms);
+		if (config.ps_params.timeout_ms) {
+			shell_fprintf(sh, SHELL_NORMAL, "PS timeout: %d ms\n",
+					config.ps_params.timeout_ms);
+		} else {
+			shell_fprintf(sh, SHELL_NORMAL, "PS timeout: disabled\n");
+		}
+
```

Comment:
I wonder if these strings could be made same here and below. Now both allocate space because they are similar but not the same.
For example if we want to print extra ":" to the string, we could have format strings like this
```
shell_fprintf(sh, SHELL_NORMAL, "PS timeout%sdisabled\n", ": ");
```
and then in below one could have
```
shell_fprintf(sh, SHELL_NORMAL, "PS timeout%sdisabled\n", " ");
```

With this change, the format string is only saved once to the flash and we can save about 22 bytes. That feels small but could matter when done consistently for the whole program.
Similar suggestion also to first print
```
shell_fprintf(sh, SHELL_NORMAL, "PS timeout%s%d ms\n", ": ",  params.timeout_ms);
```
etc

Reply:
Sure, they can be easily made the same `:` was missed in one.